OlinkAnalyze/R/olink_normalization_utils.R

```diff
@@ -1023,8 +1024,13 @@ olink_norm_input_cross_product <- function(lst_df,
             )
           ),
         by = stats::setNames(object = "OlinkID_HT", nm = l_ht_oid_rename),
-        relationship = "many-to-one"
-      )
+        relationship = "many-to-many"
```

Comment:
Makes sense why this has been changed to many-to-many. 4 rows for each 3K correlation assay. Looks good.